modules/nw-metallb-operator-custom-resources.adoc

```diff
@@ -34,7 +34,7 @@ BFD provides faster path failure detection than BGP alone provides.
 After you add the `MetalLB` custom resource to the cluster and the Operator deploys MetalLB, the MetalLB software components, `controller` and `speaker`, begin running.
 
 The Operator includes validating webhooks for the `AddressPool` and `BGPPeer` custom resources.
-The webhook for the address pool custom resource performs the following checks: that the address pool name is unique.
+The webhook for the address pool custom resource performs the following checks: 
```

Comment:
@kquinn1204 I did miss this the first time through. Apologies. While you are here, please add the `:_content-type: <type>`